akka-projection-grpc/src/main/scala/akka/persistence/EventWriter.scala

```diff
@@ -0,0 +1,337 @@
+/*
+ * Copyright (C) 2009-2023 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence
+
+import akka.actor.typed.ActorRef
+import akka.actor.typed.ActorSystem
+import akka.actor.typed.Behavior
+import akka.actor.typed.Extension
```

Comment:
this must be the classic extension, since we will place it in akka-persistence

Reply:
I actually expected to put it in akka-persistence-typed (also implemented the actual actor as typed)

Reply:
you are right, we want to write the actor in typed anyway